ehb_service/apps/api/views/organization.py

```diff
@@ -12,9 +14,8 @@
 
 log = logging.getLogger(__name__)
 
-class OrganizationQuery(Resource):
-    supported_accept_types = ['application/json', 'application/xml']
-    model = 'core.models.identities.Organization'
+@permission_classes((permissions.AllowAny,))
```

Comment:
Removed the @permissionsAllowAny header from all the classes. Created a api/authentication.py file to override Django authentication so that we can continue authenticating using the api tokens that we've created instead of switching over to Django-generated auth-tokens. 803fea7